ChangeLog.d/fix-x509-dn-unusual-string-types.txt

```diff
@@ -0,0 +1,6 @@
+Bugfix
+   * Fix the handling of unusual string types in X509 Distinguished Names
+     in mbedtls_x509_dn_gets().
+     Previously we rendered T61String, UniversalString and BMPString types
+     using numeric OID / der-encoded hex string format. Change this so that
+     all string types are provided directly as strings.
```

Comment:
Doesn't this resolve an open issue? It seems a bit weird to fix a bug in an LTS that doesn't have an open issue.

Reply:
Ah whoops, the issue was reported on the mailing list and I had forgotten to open an issue to track it.

Reply:
I have opened #9376.